pkg/authorization/api/types.go

```diff
@@ -71,6 +77,29 @@ var (
 		KubeExposedGroupName:   {"pods", "replicationcontrollers", "serviceaccounts", "services", "endpoints", "persistentvolumeclaims", "pods/log"},
 		KubeAllGroupName:       {KubeInternalsGroupName, KubeExposedGroupName, QuotaGroupName},
 		KubeStatusGroupName:    {"pods/status", "resourcequotas/status", "namespaces/status"},
+
+		OpenshiftNonEscalatingViewableGroupName: {
```

Comment:
Let's add that now `func init() { util.NewStringSet(KubeNonEscalatingViewableGroupName).Diff() }`